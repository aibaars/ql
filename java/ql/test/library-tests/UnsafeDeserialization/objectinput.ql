import default
import semmle.code.java.security.UnsafeDeserialization

from ObjectInputStreamReadObjectMethod m, MethodAccess ma
where ma.getMethod() = m
select ma, m.getDeclaringType().getName()
