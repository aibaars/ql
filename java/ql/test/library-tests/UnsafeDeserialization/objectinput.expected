| Test.java:9:3:9:27 | readObject(...) | ObjectInputStream |
| Test.java:10:3:10:26 | readObject(...) | ObjectInput |
